packages/n4s/src/enforce/compounds/__tests__/anyOf.test.js

```diff
@@ -0,0 +1,50 @@
+import enforce from 'enforce';
+import anyOf from 'anyOf';
+
+describe('AnyOf validation', () => {
+  describe('Base behavior', () => {
+    it('Should fail when all rules fail', () => {
+      expect(
+        anyOf(
+          'test',
+          enforce.isNumber(),
+          enforce.isUndefined()
+        )
+      ).toBe(false);
+    });
+    it('Should succeed when atleast one rule applies', () => {
+      expect(
+        anyOf(
+          5,
+          enforce.isString(),
+          enforce.isNumber(),
+          enforce.isUndefined()
+        )
+      ).toBe(true);
+    });
+    it('Should fail with no rules', () => {
+      expect(
+        anyOf(5)
+      ).toBe(false);
```

Comment:
Ok, let's go with that 👍

Reply:
Well I was sure I actually went the way we agreed, and it was indeed supposed to be true and rules.length above.
I'll take care of it and the typing issue first thing tomorrow 😄